home/tests/test_content_import_export.py

```diff
@@ -110,20 +104,12 @@ def strip_leading_whitespace(entry: ExpDict) -> ExpDict:
     strip_leading_whitespace,
 ]
 
-OLD_EXPORT_FILTER_FUNCS = [
-    remove_translation_tag_from_tags,
```

Comment:
I think we can remove this function (somewhere above this line) now that we don't call it anymore?